### DISCUSSION TITLE
claiming back the gv (visual-reselect)

### DISCUSSION THREAD

**Discussion by benzwt** [NvChad/NvChad#2320] [Q&A] [answered]
The default key for spawning a vertical NVterm is "leaderv"

     ["<leader>v"] = {
      function()
        require("nvterm.terminal").new "vertical"
      end,
      "New vertical term",
    },

Unfortunately, my leader key is 'g'; thus I can't use the visual-reselect.

I have no choice but to disable the mapping,
```lua
    n = {
      ["<leader>v"] = "",
    },
```
Here comes the question. How do I claim back the gv (visual-reselect) ?

thanks!

**Comment by siduck** [comment]
how is gv related to leaderv ?

**Reply by benzwt** [reply]
we leader key is 'g'.

**Reply by siduck** [reply]
@benzwt no leader key is space in nvchad

**Reply by benzwt** [reply]
'g' is my custom leader key.

**Reply by siduck** [reply, marked as answer]
@benzwt then not our issue! :p

all you can do is assign gv to that mapping and not disable it. assigning that  key something else will override properly btw
